examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -210,10 +213,11 @@ def forward(self, emission: torch.Tensor) -> str:
         indices = torch.argmax(emission, dim=-1)  # [num_seq,]
         indices = torch.unique_consecutive(indices, dim=-1)
         indices = [i for i in indices if i != self.blank]
-        return "".join([self.labels[i] for i in indices])
+        joined = "".join([self.labels[i] for i in indices])
+        return joined.replace("|", " ").lower().strip().split()
```

Comment:
Looking at `replace("|", " ")`, I wonder if we should add an argument to [`get_labels()`](https://pytorch.org/audio/main/pipelines.html#torchaudio.pipelines.Wav2Vec2ASRBundle.get_labels) to customize this. I did not add this because I was looking the labels from written language perspective and not all the languages have spaces in their writing system. (i.e. Japanese) Meanwhile, in the decoding mechanism this is called silence token and silence is common to all the spoken languages.